packages/mds-logger/index.ts

```diff
@@ -29,8 +31,9 @@ const log = (level: LogLevel, ...args: unknown[]): string[] => {
   if (redacted.length) {
     const timestamp = Date.now()
     const ISOTimestamp = new Date(timestamp).toISOString()
+    const requestId = httpContext.get('x-request-id')
```

Comment:
We don't want to log out an `undefined` if this doesn't exist, think of a better way to do that